doc/quickstart.md

```diff
@@ -113,7 +113,13 @@ True
 >>> plc.read_by_name('global.bool_value', pyads.PLCTYPE_BOOL)
 False
 >>> plc.close()
+```
+If the type is not provided, it will be read from the server before reading or writing the value
```

Comment:
We discussed it [here](https://github.com/stlehmann/pyads/pull/149#discussion_r463442242), but since I have the chance to start from fresh I will remove this from the updated PR and keep it for another PR later